docs/guides/data-binding/how-to-bind-multiple-properties.md

```diff
@@ -0,0 +1,102 @@
+﻿---
+id: how-to-bind-multiple-properties
+title: How to Bind Multiple Properties
+---
+
+## MultiBinding
+
+In scenarios where a target property must be assigned a result calculated from several other bound properties, a 
+`MultiBinding` may be the appropriate solution. `MultiBinding` aggregates multiple bindings and produces a result 
+through the use of an `IMultiValueConverter`. The `Convert` method is called each time any of the
+bound properties change.
+
+:::warning
+
+Only supports `BindingMode.OneTime` and `BindingMode.OneWay`.
+
+:::
+
+## IMultiValueConverter
+
+Similar to `IValueConverter` in that it defines conversions to a target property. There is no `ConvertBack` 
+method as aggregate operations are irreversible.
+
+```csharp
+public interface IMultiValueConverter
+{
+    object? Convert(IList<object?> values, Type targetType, object? parameter, CultureInfo culture);
+}
+```
+
+## MultiBinding Example
+
+Consider the following scenario where you have the following ViewModels:
+
+```csharp
+public partial class ColorViewModel : ViewModelBase
+{
+    [ObservableProperty] private byte _red;
+    [ObservableProperty] private byte _green;
+    [ObservableProperty] private byte _blue;
+    [ObservableProperty] private byte _alpha;
+}
+
+public partial class MainViewModel : ViewModelBase
+{
+    [ObservableProperty] private ColorViewModel _color;
+}
```

Comment:
It's a bit iffy for me to rely on SourceGenerated view models as a sample. Since this concept isn't as familiar for many folks. But it's a nitpick.

Reply:
I thought the same. But I'm not sure what the lesser evil is here: use sourcegen for brevity, use an MVVM framework with full properties (Mvvm Toolkit vs ReactiveUI are still different), or plop in an entire custom `INotifyPropertyChanged` implementation? I do need to demonstrate *something* for any example to make sense.

Reply:
Specifically with readonly view model - it can be just readonly for simplicity. But it's not always the case.

Reply:
True, but readonly eliminates the point of using a `MultiBinding`. Otherwise, using a `Binding` with a `Converter` for the full object is the right approach because the properties won't notify anyways.

Reply:
Actually, I can rework this with data coming from 4 `TextBox` or 4 `NumericUpDown` controls and create a visual. I don't think the VM choice is a big deal, but View-only has obvious merits in being self-contained (besides the converter). I'll amend when I have time.

Reply:
I think a comment which should clarify it and link to source gen docs should be enough imo

Reply:
Updated example to View-only, added visual